db-checkbook/build_scripts/02_build.py

```diff
@@ -1,8 +1,102 @@
 import pandas as pd
+import geopandas as gpd
+import re
+import os
+from pathlib import Path
 
-def run_categorization(df):
-    # 
-    return 
+_curr_file_path = Path(__file__).resolve()
+LIB_DIR = _curr_file_path / '..' / '.library'
+
+def _merge_cpdb_geoms() -> gpd.GeoDataFrame: 
+    """
+    :return: cleaned checkbook nyc data
+    :rtype: pandas df
+    """
+    def extract_year(filename):
+        # TODO: error checking
+        match = re.search(r'^\d{4}', filename)
+        if match:
+            return int(match.group())
+        return None
+    
+    file_list = [p.name for p in Path(LIB_DIR).iterdir() if p.is_file()]
```

Comment:
nice - and since LIB_DIR is now itself a Path, we can just do `LIB_DIR.iterdir()`